documentation/modules/operators/proc-changing-topic-operator-mode.adoc

```diff
@@ -0,0 +1,73 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-topic-operator.adoc
+
+[id='con-changing-topic-operator-mode-{context}']
+= (Preview) Switching between Topic Operator modes
+
+[role="_abstract"]
+You can switch between topic management modes when upgrading or downgrading Strimzi, or when using the same version of Strimzi, as long as the mode is supported for that version. 
+
+.Switching from bidirectional to unidirectional topic management mode
+
+. xref:ref-operator-unidirectional-topic-operator-feature-gate-str[Enable the `UnidirectionalTopicOperator` feature gate]. 
++
+The Cluster Operator deploys the Entity Operator with the Topic Operator in unidirectional topic management mode.
+
+. The internal topics that support the Topic Operator running in bidirectional topic management mode are no longer required, so you can delete the `KafkaTopic` resources to manage them:
++
+[source,shell,subs=+quotes]
+----
+kubectl delete $(kubectl get kt -o name | grep strimzi-store-topic) && kubectl delete $(kubectl get kt -o name | grep strimzi-topic-operator)
+---- 
++
+This command deletes the internal topics, which have names starting `strimzi-store-topic` and `strimzi-topic-operator`.
+
+. The internal topics for storing consumer offsets and transaction states must be retained in Kafka. So, you must first discontinue their management by the Topic Operator before deleting the `KafkaTopic` resources.  
++
+.. Discontinue management of the topics:
++
+[source,shell,subs=+quotes]
+----
+kubectl annotate $(kubectl get kt -o name | grep consumer-offsets) strimzi.io/managed="false" && kubectl annotate $(kubectl get kt -o name | grep transaction-state) strimzi.io/managed="false"
+----
++
+By annotating the `KafkaTopic` resources with `strimzi.io/managed=false`, you indicate that the Topic Operator should no longer manage those topics. 
+In this case, we are adding the annotation to resources for managing the internal topics with names starting `consumer-offsets` and `transaction-state`.
++
+.. Delete the `KafkaTopic` resources:
```

Comment:
Here I would explicitly say that, once unmanged, we can safely delete these KafkaTopic resources in Kubernetes, without deleting the topics inside Kafka.